src/data/padics/padic_norm.lean

```diff
@@ -298,6 +310,161 @@ theorem min_le_padic_val_rat_add {q r : ℚ}
 
 end padic_val_rat
 
+section padic_val_nat
+
+/--
+Dividing out by a prime factor reduces the padic_val_nat by 1.
+-/
+lemma padic_val_nat_of_quot {p : ℕ} [p_prime : fact p.prime] {b : ℕ} (b_nonzero : b ≠ 0) (dvd : p ∣ b) :
```

Comment:
This would be `padic_val_nat_div`. My guess is that it's even true if `b = 0` because both sides of the result will (confusingly) be `0`. That saves you an assumption.